### DISCUSSION TITLE
Ошибка компиляции для платы NodeMCU 1.0 (ESP-12E Module).

### DISCUSSION THREAD

**Discussion by tushkanius** [vvip-68/LedPanelWiFi#38] [Вопросы и ответы]
![Снимок экрана от 2023-12-09 21-29-42](https://github.com/vvip-68/LedPanelWiFi/assets/96018059/d55aa39b-d523-4499-bd7f-119ca01e6756)
Ошибка компиляции для платы NodeMCU 1.0 (ESP-12E Module).
Arduino 1.8.19
версия ядра 3.1.2 
by Daniel Garcia   Версия 3.5.0 INSTALLED
Почему ошибка компиляции?

**Comment by vvip-68** [comment]
Хрустальный шар затуманился... Волшебная лупа ослабела и не пробивает "занавеску", закрывающую текст ошибки...
Не, не получается...

**Reply by tushkanius** [reply]
Arduino: 1.8.19 (Linux), Плата:"NodeMCU 1.0 (ESP-12E Module), 80 MHz, Flash, Disabled (new aborts on oom), Disabled, All SSL ciphers (most compatible), 32KB cache + 32KB IRAM (balanced), Use pgm_read macros for IRAM/PROGMEM, 4MB (FS:2MB OTA:~1019KB), 2, v2 Lower Memory, Disabled, None, Only Sketch, 921600"











/home/zubr/snap/arduino/85/.arduino15/packages/esp8266/hardware/esp8266/3.1.2/tools/mkbuildoptglobals.py
Minimal supported version of Python is 3.7
exit status 1
Ошибка компиляции для платы NodeMCU 1.0 (ESP-12E Module).

**Reply by tushkanius** [reply]
Вдруг заметил, что и старая версия V1/13 перестала компилироваться, хотя вчера было норм.

**Reply by vvip-68** [reply]
Ну из текста выше - пишет что у вас Python старой версии. Попробуйте обновить, поставить поновее
